homeassistant/components/samsungtv/entity.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import annotations
 
-from typing import Any
+from typing import TYPE_CHECKING, Any
```

Comment:
Note: all typing changes are needed because wakeonlan now conforms to PEP 561: https://github.com/remcohaszing/pywakeonlan/pull/28